src/controllers/mapping.ts

```diff
@@ -42,7 +41,8 @@ const transform = async (req: Request, res: Response) => {
       inputJson = await v2json(bodyString);
     } else {
       if (contentType?.startsWith('text/csv')) {
-        inputJson = parseCsv(req.body);
+        console.log('Content-Type header suggests CSV input');
```

Comment:
I think you should either remove this or use `getLogger`

Reply:
Switched to getLogger()